PurchasesTests/Purchasing/RCIntroEligibilityTests.swift

```diff
@@ -0,0 +1 @@
+//  Copyright RevenueCat Inc. All Rights Reserved.
```

Comment:
Is this one an empty file? Was it intended?

Reply:
TBH, this file is not included in the `xcodeproj`, I leave it there because I didn't want to either add or remove anything. But I could remove it 👍🏼